### ISSUE TITLE
Using monitor slider prevents keypress events

### ISSUE THREAD

**Issue by AmazingMech2418** [LLK/scratch-gui#2796]
### Expected Behavior

If you use the slider to edit a variable, keys should still update the key [] pressed blocks.

### Actual Behavior

It doesn't work and ignores the key press.

### Steps to Reproduce

Go to https://llk.github.io/scratch-gui/develop/#238287060, change the variable value with the slider, and press space. The project doesn't even load on Edge. It says it crashed due to a long script.

### Operating System and Browser

Windows 10 and Chrome.

**Comment by paulkaplan** [comment]
Thanks for reporting this. It has the same root cause as https://github.com/LLK/scratch-gui/issues/2457, but on a different part of the app. I'll flag this for further discussion on the team, we are caught between keyboard accessibility on form elements vs. the way scratch works, and need to make some hard decisions 😢

**Comment by LiFaytheGoblin** [comment]
Can also be seen at 141913142
